Sources/Features/Client/BacktraceReporter.swift

```diff
@@ -96,7 +104,7 @@ extension BacktraceReporter {
         attributesProvider.set(faultMessage: faultMessage)
         let resource = try reporter.generateLiveReport(exception: exception,
                                                        attributes: attributesProvider.allAttributes,
-                                                       attachmentPaths: attachmentPaths)
+                                                       attachmentPaths: attachmentPaths + attributesProvider.attachmentPathsArray)
```

Comment:
@krml19 this is why we need `attachmentPathsArray` parameter. But I will change it to `attachmentPaths`, that is definitely a better name